docs/Developers/Git_Basics.rst

```diff
@@ -380,6 +380,11 @@ Now, to revert the file to that revision, run the command:
 Now, after the file gets reverted back to the required revision, commit the
 changes and (force)push to the remote.
 
+Sometimes while rebasing it might happen that you come across a Mid-Rebase
```

Comment:
While you rebase. You may come across mid-rebase conflicts.
Does this sound good?